src/utils/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod fs;
+pub mod string;
```

Comment:
It may be easier to make the `string` module private and then re-export `RangeArgument` and `take_lines()`. `mdbook` has a bad habit of creating deeply nested modules, which makes it annoying when you're looking for things in the docs or have to import something.

Reply:
Done.